src/plugins/workspace/public/components/workspace_initial/workspace_initial.tsx

```diff
@@ -3,184 +3,245 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import React from 'react';
-import { CoreStart } from 'opensearch-dashboards/public';
+import './workspace_use_case_card.scss';
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+import { ChromeNavControl, CoreStart } from 'opensearch-dashboards/public';
 import {
   EuiLink,
   EuiPage,
   EuiIcon,
   EuiText,
-  EuiCard,
-  EuiImage,
-  EuiPanel,
   EuiTitle,
   EuiSpacer,
+  EuiPopover,
   EuiPageBody,
   EuiFlexItem,
   EuiFlexGroup,
-  EuiPageContent,
   EuiSmallButton,
-  EuiSmallButtonEmpty,
+  EuiContextMenu,
 } from '@elastic/eui';
 import { i18n } from '@osd/i18n';
-import BackgroundLightSVG from '../../assets/background_light.svg';
-import BackgroundDarkSVG from '../../assets/background_light.svg';
-import { WORKSPACE_CREATE_APP_ID } from '../../../common/constants';
+import { BehaviorSubject } from 'rxjs';
+import { WORKSPACE_CREATE_APP_ID, WORKSPACE_LIST_APP_ID } from '../../../common/constants';
 import { useOpenSearchDashboards } from '../../../../opensearch_dashboards_react/public';
+import { WorkspaceUseCase } from '../../types';
+import { WorkspaceUseCaseCard } from './workspace_use_case_card';
+import { WorkspaceUseCaseFlyout } from '../workspace_form';
+import { navigateToWorkspacePageWithUseCase } from '../utils/workspace';
 
-export const WorkspaceInitial = () => {
+export interface WorkspaceInitialProps {
+  registeredUseCases$: BehaviorSubject<WorkspaceUseCase[]>;
+}
+
+export const WorkspaceInitial = ({ registeredUseCases$ }: WorkspaceInitialProps) => {
   const {
-    services: { application, chrome, uiSettings },
+    services: { application, chrome, workspaces, http },
   } = useOpenSearchDashboards<CoreStart>();
-  const isDashboardAdmin = application.capabilities.dashboards?.isDashboardAdmin;
-  const logos = chrome.logos;
-  const createWorkspaceUrl = application.getUrlForApp(WORKSPACE_CREATE_APP_ID, { absolute: true });
-  const settingsAndSetupUrl = application.getUrlForApp('settings_landing', { absolute: true });
-  const isDarkTheme = uiSettings.get('theme:darkMode');
-  const backGroundUrl = isDarkTheme ? BackgroundDarkSVG : BackgroundLightSVG;
+  const isDashboardAdmin = !!application.capabilities.dashboards?.isDashboardAdmin;
+  const availableUseCases = registeredUseCases$
+    .getValue()
+    .filter((item) => !item.systematic || item.id === 'all');
+  const workspaceList = workspaces.workspaceList$.getValue();
+  const [isUseCaseFlyoutVisible, setIsUseCaseFlyoutVisible] = useState(false);
+  const [defaultUseCaseId, setDefaultUseCaseId] = useState(availableUseCases[0].id);
```

Comment:
```suggestion
  const [defaultExpandedUseCaseId, setDefaultExpandedUseCaseId] = useState(availableUseCases[0].id);
```
Nit: defaultUseCaseId seems a little bit confusing.

Reply:
Sure, I will update it.